modules/odr_api/odr_api/main.py

```diff
@@ -31,6 +31,7 @@ def test_communication():
 app.include_router(annotation_router, prefix=settings.API_V1_STR)
 app.include_router(auth_router, prefix=settings.API_V1_STR)
 app.include_router(embedding_router, prefix=settings.API_V1_STR)
+app.include_router(health.router)
```

Comment:
Try and match to the existing routes. Should have a prefix and same naming style